cloud.profiles.d/edx.yml

```diff
@@ -0,0 +1,21 @@
+edx:
```

Comment:
Should move this to the `orchestrate/aws/cloud_profiles` directory (I may be misremembering the proper path).